ios/FluentUI/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -108,6 +132,15 @@ public class ControlTokenSet<T: TokenSetKey>: ObservableObject {
 
     /// Reference to the default value lookup function for this control.
     private var defaults: ((_ token: T, _ theme: FluentTheme) -> ControlTokenValue)?
+
+    /// Holds the sink for any changes to the control token set.
+    private var changeSink: AnyCancellable?
+
+    // Stores the notification handler for .didChangeTheme notifications.
```

Comment:
Nit: Extra slash here?

Reply:
Extra nit the commenting style is a bit inconsistent throughout wasn't sure if that was intentional or not

Reply:
Good call - the goal should be two slashes for comments in a method, and three for documentation of a variable or function. Like so:

```swift
/// Returns a random number
func veryRandomNumber() -> Int {
    // The number four was randomly selected by rolling a six-sided die.
    return 4
}
```